crates/runtime/src/sources/solana_accounts_rpc.rs

```diff
@@ -0,0 +1,157 @@
+                            // filters: Some(vec![RpcFilterType::Memcmp(Memcmp::new(offset, encoded_bytes))]),
+                            filters: None,
+                            account_config: RpcAccountInfoConfig {
+                                encoding: None,
+                                data_slice: None,
+                                commitment: None, // Already set in the client
+                                min_context_slot: None,
+                            },
+                            with_context: Some(false),
+                            sort_results: Some(false),
+                        })
+                        .await
+                        .map_err(|e| {
+                            crate::Error::Io(std::io::Error::new(
+                                std::io::ErrorKind::Other,
+                                e.to_string(),
+                            ))
+                        })?;
+
+                    for (acc_pubkey, account) in accounts {
+                        let update = SubscribeUpdate {
+                            filters: vec![filter_id.clone()],
+                            update_oneof: Some(UpdateOneof::Account(SubscribeUpdateAccount {
+                                account: Some(SubscribeUpdateAccountInfo {
+                                    pubkey: acc_pubkey.as_array().to_vec(),
+                                    lamports: account.lamports,
+                                    owner: account.owner.as_array().to_vec(),
+                                    executable: account.executable,
+                                    rent_epoch: account.rent_epoch,
+                                    data: account.data,
+                                    write_version: 0,
+                                    txn_signature: None,
+                                }),
+                                slot: 0,
+                                is_startup: true,
+                            })),
+                        };
+
+                        let res = tx.send(Ok(update)).await;
+                        if res.is_err() {
+                            tracing::error!("Failed to send update to buffer");
+                        }
+                    }
+                }
+            }
+        }
+
+        Ok(())
+    }
+
+    fn set_filters_unchecked(&mut self, filters: Filters) { self.filters = Some(filters); }
+
+    fn set_config_unchecked(&mut self, config: YellowstoneConfig) {
+        self.config = Some(SolanaAccountsRpcConfig {
+            endpoint: config.endpoint,
+            timeout: config.timeout,
+        });
+    }
+
+    fn get_filters(&self) -> &Option<Filters> { &self.filters }
+
+    fn get_config(&self) -> Option<YellowstoneConfig> {
+        self.config.clone().map(
+            |SolanaAccountsRpcConfig { endpoint, timeout }| YellowstoneConfig {
+                endpoint,
+                timeout,
+                x_token: None,
+            },
```

Comment:
```suggestion
Into::into
```

See can represent in from trait